process-dataset/steps/write-per-cell-jsons.js

```diff
@@ -35,19 +35,17 @@ const formatAndWritePerCellJsons = async (readFolder, outFolder, featureDataFile
                     acc[FILE_INFO_KEYS[index]] = cur;
                     return acc;
                 }, {});
-                const cellLine = find(cellLines, {
-                    [CELL_LINE_DEF_NAME_KEY]: fileInfo[CELL_LINE_NAME_KEY]
-                });
-                if (!cellLine) {
-                    console.error("No matching cell line for this cell in the dataset", fileInfo)
+                const groupBy = find(featureDefs, {key: defaultGroupBy}).options[fileInfo.groupBy]
```

Comment:
probably can make this code more readable. Ie, make it clear what is a key and what is a value

Reply:
`groupingValue`? `categoryData` (`name`, `color`, `key`).

Reply:
if cell line is just a feature, then the groupby grouping value could just come from the feature data and doesn't need to be in the fileInfo.  what is the best reason to keep it in there?

Reply:
my thinking was this will make it easier to make these updates without the app ever being completely broken, and without having to add a lot of extra code to the front end that we then remove. But maybe downtime isn't a huge concern because we don't have a ton of users, so we can update the front end and the database at the same time